eclair-rpc/src/main/scala/org/bitcoins/eclair/rpc/client/EclairRpcClient.scala

```diff
@@ -699,10 +699,13 @@ class EclairRpcClient(val instance: EclairInstance, binary: Option[File] = None)
       }
     }
 
-    val started = AsyncUtil.retryUntilSatisfiedF(() => isStarted,
-                                                 duration = 1.seconds,
-                                                 maxTries = 60)
-
+    val started: Future[EclairRpcClient] = {
+      for {
+        _ <- AsyncUtil.retryUntilSatisfiedF(() => isStarted,
+                                            duration = 1.seconds,
+                                            maxTries = 60)
+      } yield this.asInstanceOf[EclairRpcClient]
```

Comment:
I think you can just return `this` here? (remove `.asInstanceOf[EclairRpcClient]`)?